### ISSUE TITLE
Is this project still maintained?

### ISSUE THREAD

**Issue by ballPointPenguin** [krakenjs/kraken-js#472]
There seems to have been a sharp drop-off in activity here and I'm wondering if I should use this or not on a new project. Is there any official word on the status of kraken development?

**Comment by tlivings** [comment]
Here is a good thread on this topic: https://github.com/krakenjs/kraken-js/issues/414

Short version: this module has reached a steady state that doesn't demand, currently, a lot of iteration. Most activity occurs in supporting modules.

**Comment by tlivings** [comment]
As a sub note: we still use Kraken internally with over 130 applications in production and growing. Over 300 millions requests per day on average.

**Comment by grawk** [comment]
There will be some new examples, documentation, and modules out soon around usage of react/webpack. We also plan to reboot our krakenjs.com website soon.

**Comment by ballPointPenguin** [comment]
Thank you. I am happy to hear this.

tl;dr Yes

**Comment by jasisk** [comment]
Unquestionably, a number of the kraken (org, not module) repos need a bit of spit-shine. The team has been stretched a bit thin and, speaking personally, I had to take a bit of a break from things due to some personal life changes. While I'm with @tlivings on the relatively feature-completeness of kraken (module, not org), issues and pull-requests _have_ gotten stale—a lot of that is on me.

Hopefully we can clean some of that up soon which—in addition to the updates @grawk pointed out—_should_ help in fixing the questionable perception.
